docs/deploy_logging.md

```diff
@@ -0,0 +1,49 @@
+# Deploy openshift-logging on cluster
+
+### Pre-requisites before starting to deploy openshift-logging
+
+###### It is recommended to have minimum m5.xlarge setup.
+
+The images for the openshift-logging need to be pulled from the quay,
+sync the images with the internal registry and update the cluster with the new images for containers.
+
+
+Note: Steps are to be followed till OCP 4.2 is GAed, These steps can be skipped after that
+
+1. Create a quay.io account [https://quay.io]
+
+2. Add your quay username in the sheet given below and wait for the invite sent
+from quay.io ocp-repo maintainers
+https://docs.google.com/spreadsheets/d/1OyUtbu9aiAi3rfkappz5gcq5FjUbMQtJG4jZCNqVT20/edit#gid=0
+
+3. Generate the quay_token with the command
+
+```
+curl -sH "Content-Type: application/json" -XPOST https://quay.io/cnr/api/v1/users/login -d '
+{
+    "user": {
+        "username": "'"$USER"'",
+        "password": "'"$PASSWD"'"
+    }
+}' | jq -r '.token'
+```
+
+4.Save the token as quay.token under ocs-ci/data/
+
+5.Now install podman in your local using https://github.com/containers/libpod/blob/master/install.md
+
+6.To give root privileges for your podman to run, 
+add your username ex:sraghave to /etc/subuid with the exact same mapping as user (sraghave:100000:65536)
+
+Example:-
+
+$ cat /etc/subuid
+
+sraghave:100000:65536
+
+7. Then run the script to deploy openshift-logging on your using
+```run-ci --no-print-logs --cluster-path /home/username/ tests/e2e/logging/test_openshift-logging.py```
+
+The script is currently under review, will soon be merged.
+######Note:
+The openshift-logging deployment will soon be moved along with ocs-ci deployment enabling the user to set an option true/false in default_config.yaml
```

Comment:
file surround with \` . `defautl_config.yaml` but this will needs to be set in customer config, or we can create config in `conf/ocsci/deployment_logging.yaml` which will contain the value which will enable logging and user can pass just with `--ocsci-conf` parameter